app/models/user.rb

```diff
@@ -17,6 +17,10 @@ class User < ApplicationRecord
   # -- Callbacks ------------------------------------------------------------
   # -- Validations ----------------------------------------------------------
   # -- Scopes ---------------------------------------------------------------
+  # Preferred notification frequencies
+  scope :dailies, -> { where('preferences LIKE "%digest_frequency: daily%"') }
+  scope :instants, -> { where('preferences LIKE "%digest_frequency: instant%"') }
```

Comment:
```suggestion
  scope :digests_instant, -> { where('preferences LIKE "%digest_frequency: instant%"') }
```

Just using dailies, and instant makes me think we're looking for daily or instant Users, as they are scopes on Users. Being slightly more specific is a bit nicer.